src/test/java/org/pentaho/di/trans/steps/mongodbinput/MongoDbInputTest.java

```diff
@@ -1,19 +1,19 @@
 /*!
-* Copyright 2010 - 2019 Hitachi Vantara.  All rights reserved.
-*
-* Licensed under the Apache License, Version 2.0 (the "License");
-* you may not use this file except in compliance with the License.
-* You may obtain a copy of the License at
-*
-* http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing, software
-* distributed under the License is distributed on an "AS IS" BASIS,
-* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-* See the License for the specific language governing permissions and
-* limitations under the License.
-*
-*/
+ * Copyright 2010 - 2019 Hitachi Vantara.  All rights reserved.
```

Comment:
Update this copyright date to 2010 - 2021.  Current practice is that if a file contains a copyright date and gets updated, we edit the date range to include the current year.